app/src/androidTest/java/com/example/sharingang/MapActivityTest.kt

```diff
@@ -0,0 +1,67 @@
+package com.example.sharingang
```

Comment:
Nitpick: This file has ostensibly not been formatted. I'd recommend putting this through your IDE's formatter.